test/rfc3962-vectors.config

```diff
@@ -0,0 +1,50 @@
+%% -*- mode: erlang; erlang-indent-level: 4; indent-tabs-mode: nil -*-
```

Comment:
I'm assuming this file is left untouched except for the occasional static analysis -related fix. Is my assumption correct?

Reply:
I don't know... is it? This is test data.